requirements-devenv.txt

```diff
@@ -1,5 +1,6 @@
 -r requirements-linting.txt
 -r requirements-testing.txt
+ipdb
```

Comment:
If we, in fact, don't use `ipdb` anywhere, then I don't think it makes sense to list it as a dependency here, either. Developers who find `ipdb` helpful can install the package themselves.

```suggestion
```